services/datcore-adapter/src/simcore_service_datcore_adapter/utils/app_data.py

```diff
@@ -0,0 +1,51 @@
+"""
+    AppData refers to instances of dataclasses stored in the FastApi application
+"""
+from fastapi import FastAPI
+
+
+class AppDataMixin:
```

Comment:
Is it not possible to use something from `abc` module to avoid all the below asserts? Just wondering.

Reply:
this was taken from other FastApi applications that we use. I would say we would first need a library. now probably @pcrespov nows better.